VolleyManagement.UI/spa/index.html

```diff
@@ -1,14 +1,21 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>VolleyManagement.UI</title>
   <base href="/">
 
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <meta name="viewport"
+        content="width=device-width, initial-scale=1">
+  <link rel="icon"
+        type="image/x-icon"
+        href="favicon.ico">
 </head>
+
 <body>
-  <vm-app metadatafile="test-tournament.json"></vm-app>
+  <vm-app></vm-app>
+  <vm-app-integration metadatafile="test-tournament.json"></vm-app-integration>
```

Comment:
"If we remove "vm-app" or "vm-app-integration" angular bootstrap mechanism will not handle both cases, because on dev start it will not find root component tag. I put them both to allow us switch between scenarious without any other specific actions"

So we have to keep it here even if it will not be used? Will Angular try to render it?

Reply:
Due ti my experiments no)

Reply:
So we can't have two different index.html files one for regular another for VOU apps?